docs/upgrading.rst

```diff
@@ -1,5 +1,36 @@
 Upgrading
 =========
+Docker-compose
+--------------
+When you deploy a vanilla docker-compose, it will create a persistent volume for your MySQL database. As long as your volume is there, you should not lose any data.
+
+Using docker images provided in DockerHub
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+   If you're using ``latest``, then you need to pre pull the ``latest`` from DockerHub to update.
+
+The generic upgrade method for docker-compose follows these steps:
+- Pull the latest or tagged images from https://hub.docker.com/repository/docker/defectdojo/defectdojo-django and https://hub.docker.com/repository/docker/defectdojo/defectdojo-nginx
+- Go to the directory where your docker-compose.yml file lives
```

Comment:
I do 
- Go to the directory where your docker-compose.yml file lives
- `docker-compose pull`

Reply:
did an update to make it more specific. wdyt @madchap

Reply:
In rst, backward single quotes need to be doubled to provide the `code` style (contrary to one quote for markdown).
The indentation seems to throw rst warnings too.

Reply:
When are we moving to github pages or something not as crappy as RST?